src/Parser.php

```diff
@@ -33,6 +33,8 @@ public function __construct($sText, ?Settings $oParserSettings = null, $iLineNo
      * Sets the charset to be used if the CSS does not contain an `@charset` declaration.
      *
      * @param string $sCharset
+     *
+     * @internal
      */
     public function setCharset($sCharset): void
```

Comment:
The description suggests that this is not an internal method.  However, the example in the readme shows setting the charset via `Settings::withDefaultCharset()`.  But searching the code base, I find this is not actually called internally (internally we set the charset directly on the `ParserState` object).

So if we don't want to expose this method moving forwards, we should mark it as deprecated.  In which case, we want the logger inferface implementation before we can do so, don't we?

Reply:
They are now deprecated (#688) instead.